go.mod

```diff
@@ -1,10 +1,31 @@
 module github.com/nix-community/go-nix
 
-go 1.15
+go 1.17
 
 require (
 	github.com/alecthomas/kong v0.5.0
 	github.com/dgraph-io/badger/v3 v3.2103.2
 	github.com/google/go-cmp v0.5.5
 	github.com/stretchr/testify v1.7.0
 )
+
+require (
```

Comment:
This mess was all produced by `go mod tidy`.

Reply:
So tidy! :facepalm: